libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/logger/SalesforceLogger.java

```diff
@@ -49,8 +52,10 @@
 public class SalesforceLogger {
 
     private static final String TAG = "SalesforceLogger";
-    private static final String LOG_LINE_FORMAT = "LEVEL: %s, TAG: %s, MESSAGE: %s";
-    private static final String LOG_LINE_FORMAT_WITH_EXCEPTION = "LEVEL: %s, TAG: %s, MESSAGE: %s, EXCEPTION: %s";
+    private static final String LOG_LINE_FORMAT = "TIME: %s, LEVEL: %s, TAG: %s, MESSAGE: %s";
+    private static final String LOG_LINE_FORMAT_WITH_EXCEPTION = "TIME: %s, LEVEL: %s, TAG: %s, MESSAGE: %s, EXCEPTION: %s";
+    private static final String US_DATE_FORMAT = "yyyy.MM.dd G 'at' HH:mm:ss z";
```

Comment:
Could we add the milliseconds (HH:mm:ss.SSS)?
Also should we just mirror the logcat date/time format e.g.
05-30 15:18:15.839 ?